src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/CombinationController.php

```diff
@@ -117,12 +136,41 @@ public function updateCombinationFromListingAction(int $combinationId, Request $
         return $this->json([]);
     }
 
+    /**
+     * @param AttributeGroup[] $attributeGroups
+     *
+     * @return array<int, array<string, mixed>>
+     */
+    private function formatAttributeGroupsForPresentation(array $attributeGroups): array
+    {
+        $contextLangId = $this->getContextLangId();
+
+        $formattedGroups = [];
+        foreach ($attributeGroups as $attributeGroup) {
+            $attributes = [];
+            foreach ($attributeGroup->getAttributes() as $attribute) {
+                $attributes[] = [
+                    'id' => $attribute->getAttributeId(),
+                    'name' => $attribute->getLocalizedNames()[$contextLangId],
```

Comment:
```suggestion
                    'name' => $attribute->getLocalizedNames()[$contextLangId] ?? '',
```

Reply:
If we handle a fallback, I'd rather use the default language id (if present) and if not use the first one in the list, so that we always have a name returned

Reply:
A specification explains anywhere the fallback ?

Reply:
I'd say, of course not 😅 Although there are very small chances that the context language is not present in the whole list of languages.. If not completely impossible It's not like a former language associated to an Order that would have been deleted (like the bug Thomas had recently)

The language in the context is necessarily among the installed languages, so anyway this extra safety will probably be useless anyway

Reply:
I think the implementation should be : $attribute->getLocalizedName($contextLangId) which returns a string (empty or not)